atmos.yaml

```diff
@@ -17,6 +17,16 @@
 # are considered paths relative to 'base_path'.
 base_path: "./examples/quick-start-advanced"
```

Comment:
Looks like we had a preexisting bad copy/paste.

```suggestion
base_path: "./"
```

Reply:
resolved